tests/Core/Content/Product/SalesChannel/Listing/ProductSearchRouteTest.php

```diff
@@ -5,14 +5,32 @@
 namespace FINDOLOGIC\FinSearch\Tests\Core\Content\Product\SalesChannel\Listing;
 
 use FINDOLOGIC\FinSearch\Core\Content\Product\SalesChannel\Search\ProductSearchRoute;
+use FINDOLOGIC\FinSearch\Tests\Traits\DataHelpers\ProductHelper;
+use FINDOLOGIC\FinSearch\Utils\Utils;
+use OpenApi\Util;
 use PHPUnit\Framework\MockObject\MockObject;
+use ReflectionObject;
+use Shopware\Core\Content\Product\ProductEntity;
 use Shopware\Core\Content\Product\SalesChannel\Search\AbstractProductSearchRoute;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityCollection;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\EntitySearchResult;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
+use Shopware\Core\Framework\Test\TestCaseBase\SalesChannelFunctionalTestBehaviour;
 
 class ProductSearchRouteTest extends ProductRouteBase
 {
-    /** @var AbstractProductSearchRoute|MockObject  */
+    use SalesChannelFunctionalTestBehaviour;
+    use ProductHelper;
+
+    /** @var AbstractProductSearchRoute|MockObject */
     private $original;
 
+    /**
+     * @var MockObject|\Shopware\Core\System\SalesChannel\SalesChannelContext
```

Comment:
`\Shopware\Core\System\SalesChannel\SalesChannelContext` can be imported. I also prefer inline docblock instead of the three lines docblock, which is unnecessary bloat.